BankManagerConsoleApp/Model/Bank/BankClerk.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BankClerk.swift
+//  BankManagerConsoleApp
+//
+//  Created by 우롱차, 민성 on 2022/04/28.
+//
+
+import Foundation
+
+struct BankClerk {
+    private enum BankClerkConstant {
```

Comment:
BankClerk 내부에 작성되어 있는 Constant이며 private으로 접근제어가 되고 있기 때문에 네이밍에 BankClerk가 중복해서 들어가지 않아도 될 것 같습니다.
```suggestion
    private enum Constant {
```

Reply:
앗 감사합니다.